src/SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -55,6 +55,7 @@ public static SonarQubeIssueResolutionState ParseResolutionState(string resoluti
         {
             switch (resolution)
             {
+                case "": // TODO - check whether an empty string is a valid response
```

Comment:
The web service is returning "" in some cases - not sure if this is expected.

Reply:
I have double checked with Scanner Team and this is expected when the issue is not resolved (i.e. not `FP`, nor `Won't fix` nor `Resolved`).

Reply:
Great, thanks for checking.